idseq_dag/steps/combine_taxon_counts.py

```diff
@@ -18,10 +19,12 @@ def run(self):
         with_dcr = all(input_f.endswith("_with_dcr.json") for input_f in input_files)
         if not with_dcr:
             assert DAG_SURGERY_HACKS_FOR_READ_COUNTING
-            input_files = [input_f.replace(".json", "_with_dcr.json") for input_f in input_files]
```

Comment:
I'm wondering if the fix here should be to just remove this line. It seems safer than renaming the files on disk and will have the same effect in `combine_counts`.

Reply:
I agree but it must be important if it's in there